src/main.js

```diff
@@ -122,7 +126,8 @@ makeYourOwnPosterButton.addEventListener('click', showFormView);
 savedPostersButton.addEventListener('click', showSavedPostersView);
 takeMeBackButton.addEventListener('click', sendToMainPage);
 backToMainButton.addEventListener('click', sendToMainPage);
-
+// showNewPosterButton.addEventListener('click' , createNewPoster);
+inputForm.addEventListener('submit', createNewPoster);
```

Comment:
@Estaffieri - Is this comment important/needed? Do we need to keep it in or should we clear it out?

Reply:
Let me test which we need and I will edit as required.

Reply:
Yep! Looks like we need to delete that event listener. I'll make that change.